libs/common-services/src/main/java/uk/gov/di/ipv/core/library/domain/ReplayRequest.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.di.ipv.core.library.domain;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+@Builder
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class ReplayRequest {
+    @JsonProperty("Items")
+    private List<ReplayItem> items;
+
+    @JsonProperty("Count")
+    private Integer count;
+
+    @JsonProperty("ScannedCount")
+    private Integer scannedCount;
```

Comment:
What are the `Count` and `ScannedCount` fields for? (Are they just leftovers from the scan output?)

Reply:
Leftovers from the scan output - we don't strictly need these, but have left them in as might be useful for debugging